sycl/test/extensions/annotated_usm/annotated_usm_negative.cpp

```diff
@@ -0,0 +1,87 @@
+  TEST((malloc_annotated<int>), N, q, alloc::device, properties{conduit, foo{foo_enum::a}, foz{0, 1}})
+}
+
+// Expect error when the property list contain invalid comile-time property
+// Note that two errors are raised for each case, during:
+// 1. validating malloc input properties
+// 2. validating annotated_ptr properties
+void testInvalidCompileTimeProperty(sycl::queue &q) {
+  // expected-error-re@sycl/ext/oneapi/experimental/common_annotated_properties/properties.hpp:* {{static assertion failed due to requirement {{.+}}: Property is invalid for the given type.}}
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Found invalid compile-time property in the property list.}}
+  TEST(malloc_device_annotated, N, q, properties{conduit, bar})
+
+  // expected-error-re@sycl/ext/oneapi/experimental/common_annotated_properties/properties.hpp:* {{static assertion failed due to requirement {{.+}}: Property is invalid for the given type.}}
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Found invalid compile-time property in the property list.}}
+  TEST(malloc_shared_annotated, N, q, properties{conduit, baz<1>})
+
+  // expected-error-re@sycl/ext/oneapi/experimental/common_annotated_properties/properties.hpp:* {{static assertion failed due to requirement {{.+}}: Property is invalid for the given type.}}
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Found invalid compile-time property in the property list.}}
+  TEST(malloc_host_annotated, N, q, properties{conduit, boo<double>})
+
+  // expected-error-re@sycl/ext/oneapi/experimental/common_annotated_properties/properties.hpp:* {{static assertion failed due to requirement {{.+}}: Property is invalid for the given type.}}
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_util.hpp:* {{static assertion failed due to requirement {{.+}}: Found invalid compile-time property in the property list.}}
+  TEST(malloc_annotated, N, q, alloc::device, properties{conduit, bar, baz<1>})
+}
+
+void testMissingUsmKind(sycl::queue &q) {
+  // missing usm kind in property list when it is required
+  properties InP{};
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_base.hpp:* {{static assertion failed due to requirement {{.+}}: USM kind is not specified. Please specify it as an argument or in the input property list.}}
+  TEST(malloc_annotated, N, q, InP)
+  // expected-error-re@sycl/ext/oneapi/experimental/annotated_usm/alloc_base.hpp:* {{static assertion failed due to requirement {{.+}}: USM kind is not specified. Please specify it as an argument or in the input property list.}}
+  TEST((malloc_annotated<int>), N, q, InP)
+}
+
+
+void testConfilictingUsmKind(sycl::queue &q) {
```

Comment:
Typo in the name of this function, should be testConflictingUsmKind.

Reply:
typo fixed